theme_loftspace/views/images_content.xml

```diff
@@ -145,6 +145,21 @@ Check in primary_variables.scss and theme.scss -->
     <field name="name">website.s_carousel_default_image_3</field>
     <field name="url">/theme_loftspace/static/src/img/snippets/s_carousel_3.jpg</field>
 </record>
+<record id="s_carousel_intro_default_image_1" model="theme.ir.attachment">
+    <field name="key">website.s_carousel_intro_default_image_1</field>
+    <field name="name">website.s_carousel_intro_default_image_1</field>
+    <field name="url">/theme_loftspace/static/src/img/snippets/s_banner.jpg</field>
+</record>
+<record id="s_carousel_intro_default_image_2" model="theme.ir.attachment">
+    <field name="key">website.s_carousel_intro_default_image_2</field>
+    <field name="name">website.s_carousel_intro_default_image_2</field>
+    <field name="url">/theme_loftspace/static/src/img/snippets/s_product_catalog.jpg</field>
+</record>
+<record id="s_carousel_intro_default_image_3" model="theme.ir.attachment">
+    <field name="key">website.s_carousel_intro_default_image_3</field>
+    <field name="name">website.s_carousel_intro_default_image_3</field>
+    <field name="url">/theme_loftspace/static/src/img/snippets/s_carousel_1.jpg</field>
+</record>
 <record id="s_media_list_default_image_1" model="theme.ir.attachment">
     <field name="key">website.s_media_list_default_image_1</field>
     <field name="name">website.s_media_list_default_image_1</field>
```

Comment:
Controller out of the view here: 
![image](https://github.com/user-attachments/assets/7a6f9a19-5c8a-472a-b5fb-af112da6da5f)

Reply:
Same than https://github.com/odoo/design-themes/pull/893#discussion_r1766467569 :)